tests/test_backend.py

```diff
@@ -121,12 +137,12 @@ def test_successful_download(self, mock_requests, mock_load, mock_open, mock_mak
 
 
 class TestWaitForStatus(TestCase):
-    @patch('threading.Thread.start')
-    def setUp(self, mock_thread) -> None:
+    @patch("threading.Thread.start", new=lambda x: None)
+    def setUp(self) -> None:
         model = MagicMock(spec=GraphModule)
         self.runner = Runner(model, None)
```

Comment:
it looks like this step is common in many Test case classes. We don't need to do this right now, but it would be better to move this to a parent class.

Reply:
You're right. I can do that now